src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonListener.cs

```diff
@@ -116,7 +118,7 @@ private void OnLockTimer(object sender, ElapsedEventArgs e)
 
         internal async Task TryAcquireLock()
         {
-            _lockHandle = await _singletonManager.TryLockAsync(_lockId, null, _attribute, CancellationToken.None, retry: false);
+            _lockHandle = await _singletonManager.TryLockAsync(_lockId, _hostMachineId, null, _attribute, CancellationToken.None, retry: false);
```

Comment:
>_hostMachineId [](start = 72, length = 14)

Passing the host machine id straight through as the proposed lease id raises some questions:
- do they have the same naming rules. valid character sets, etc? 
- do they have the same casing rules?